scriptmodules/libretrocores/lr-mame2003.sh

```diff
@@ -63,15 +63,18 @@ function configure_lr-mame2003() {
 
     mkUserDir "$biosdir/$dir_name"
     mkUserDir "$biosdir/$dir_name/samples"
-    
+
     # copy hiscore.dat and cheat.dat
     cp "$md_inst/metadata/"{hiscore.dat,cheat.dat} "$biosdir/$dir_name/"
     chown $user:$user "$biosdir/$dir_name/"{hiscore.dat,cheat.dat}
     
     # lr-mame2003-plus also has an artwork folder
-    [[ "$md_id" == "lr-mame2003-plus" ]] && mkRomDir "$biosdir/$dir_name/artwork"
-    [[ "$md_id" == "lr-mame2003-plus" ]] && cp "$md_inst/metadata/artwork/*.*" "$biosdir/$dir_name/artwork/"
-    [[ "$md_id" == "lr-mame2003-plus" ]] && chown $user:$user "$biosdir/$dir_name/artwork/*.*"
+    if (( "$md_id" == "lr-mame2003-plus" ))
+    then
+        mkRomDir "$biosdir/$dir_name/artwork"
+        cp "$md_inst/metadata/artwork/*" "$biosdir/$dir_name/artwork/"
+        chown $user:$user "$biosdir/$dir_name/artwork/*.*"
```

Comment:
here you should also just use `*` instead of `*.*` :)

Reply:
Thanks @hhromic !